configure

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+
+AUTO_QMAIL=/var/qmail
+
+for arg in $*; do
```

Comment:
I suggest using `for arg; do` which does the same as `for arg in "$@"; do`, which helps in case any argument has a space in it, which is not expected anyway, so I am ok with `for arg in $*; do` nontheless.